doc/snippets/Microsoft.Data.SqlClient/SqlConnectionStringBuilder.xml

```diff
@@ -1132,7 +1132,7 @@
     </MinPoolSize>
     <MultipleActiveResultSets>
       <summary>
-        When true, an application can maintain multiple active result sets (MARS). When false, an application must process or cancel all result sets from one batch before it can execute any other batch on that connection. For more information, see <see href="https://msdn.microsoft.com//library/cfa084cz.aspx">Multiple Active Result Sets (MARS)</see>.
+        When true, an application can maintain multiple active result sets (MARS). When false, an application must process or cancel all result sets from one batch before it can execute any other batch on that connection. For more information, see <see href="https://learn.microsoft.com/dotnet/framework/data/adonet/sql/multiple-active-result-sets-mars">Multiple Active Result Sets (MARS)</see>.
```

Comment:
```suggestion
        When true, an application can maintain multiple active result sets (MARS). When false, an application must process or cancel all result sets from one batch before it can execute any other batch on that connection. For more information, see <see href="https://learn.microsoft.com/sql/connect/ado-net/sql/multiple-active-result-sets-mars">Multiple Active Result Sets (MARS)</see>.
```